packages/hyperion-autologging/src/ALElementInfo.ts

```diff
@@ -46,6 +47,10 @@ export default class ALElementInfo {
     return ALElementInfo.get(element) ?? new ALElementInfo(element);
   }
 
+  static getReactComponentData(element: Element, bailTraversalFunc?: BailTraversalFunc): ReactComponentData | null {
+    return ALElementInfo.get(element)?.getReactComponentData() ?? getReactComponentData_THIS_CAN_BREAK(element, bailTraversalFunc);
+  }
```

Comment:
great, now all you need to know if first part didn't work, and second part didn't return `isTruncated`, then you can call `.getOrCreate()` function and update the `this.reactComponentData` field for it. (see bellow).